lib/bit_array.h

```diff
@@ -96,4 +96,20 @@ int bit_array_set_bit( bit_array * this, size_t index, bool bit_value );
  */
 int bit_array_pop_bit( bit_array * this );
 
+/**
+ * Combines two bit_arrays and returns a copy containing the contents of both bit_arrays together.
```

Comment:
Can you specify how they get combined? This may not be apparent to users

Reply:
done